### ISSUE TITLE
Bugs in the programme "current_law_policy.json"

### ISSUE THREAD

**Issue by shekharkhetan** [TPRU-demo/pitaxcalc-demo#56]
Many of the CPI inflatable variables in rupee denominated  policy was set as  false and should be corrected .

Here is the current_law_policy.json file message lines that caused the issue 
```````````
"_rebate_thd": {
        "long_name": "rebate under section 87A threshold)",
        "description": "Income below this threshold is eligible for the rebate.",
        "itr_ref": "ITR-1 instructions, Tax Computation Table",
        "notes": "",
        "row_var": "AYEAR",
        "row_label": ["2017"],
        "start_year": 2017,
        **"cpi_inflatable": false,**
        "cpi_inflated": false,
        "col_var": "",
        "col_label": "",
        "boolean_value": false,
        "integer_value": false,
        "value": [500000],
        "range": {"min": 0, "max": 9e99},
        "out_of_range_minmsg": "",
        "out_of_range_maxmsg": "",


````````````

**Comment by shekharkhetan** [comment]
This is the bug report I got:
````````````
Traceback (most recent call last):
  File "app14.py", line 17, in <module>
    reform = Calculator.read_json_param_objects('app14_reform.json', None)
  File "C:\Users\hp\Desktop\wb training-sept18\pitaxcalc-demo\taxcalc\calculator.py", line 731, in read_json_param_objects
    rpol_dict = Calculator._read_json_policy_reform_text(txt)
  File "C:\Users\hp\Desktop\wb training-sept18\pitaxcalc-demo\taxcalc\calculator.py", line 951, in _read_json_policy_reform_text
    prdict = Calculator._convert_parameter_dict(tdict)
  File "C:\Users\hp\Desktop\wb training-sept18\pitaxcalc-demo\taxcalc\calculator.py", line 1054, in _convert_parameter_dict
    year = int(skey)
ValueError: invalid literal for int() with base 10: 'cpi_inflated'
```````````````````

**Comment by martinholmer** [comment]
@shekharkhetan, Thanks for your bug report in issue #56 concerning inaccurate values in the `current_law_policy.json` file of the `cpi_inflatable` key.  I plan to develop a pull request on Monday (after your Monday workday is over) that fixes the problems you identify in issue #56 along with the issues identified in issue #55.

@indira-tpru @adc2-tpru @keshavchoudhary87 @sebastiansajie

**Comment by martinholmer** [comment]
See pull request #60 for a proposed resolution for issues #55 and #56.

@indira-tpru @adc2-tpru @keshavchoudhary87 @sebastiansajie

**Comment by martinholmer** [comment]
Merge of pull request #60 resolves issue #56.
